lib/rdoc/web_ri.rb

```diff
@@ -0,0 +1,170 @@
+# frozen_string_literal: true
+require 'open-uri'
+require 'nokogiri'
```

Comment:
I don't think we can assume users would have these gems installed?

Reply:
Agreed (for `nokogiri`);  I think `open-uri` is installed with Ruby.  But, better to avoid both.